modules/ROOT/pages/tutorial-user-systemd-unit-on-boot.adoc

```diff
@@ -47,3 +113,63 @@ storage:
       target: /home/sleeper/.config/systemd/user/linger-example.service
       hard: false
 ----
+
+This config can be converted to Ignition:
+
+[source,bash]
+----
+fcct --pretty --strict fcct-user.yaml --output user.ign
+----
+
+== Testing
+
+Just as before we will use the following to boot the instance:
+
+[source,bash]
+----
+# Setup the correct SELinux label to allow access to the config
+chcon --verbose --type svirt_home_t user.ign
+
+# Start a Fedora CoreOS virtual machine
+virt-install --name=fcos --vcpus=2 --ram=2048 --os-variant=fedora-coreos-stable \
+    --import --network=bridge=virbr0 --graphics=none \
+    --qemu-commandline="-fw_cfg name=opt/com.coreos/config,file=${PWD}/user.ign" \
+    --disk=size=20,backing_store=${PWD}/fedora-coreos.qcow2
+----
+
+We can then verify that the unit has been started under the sleeper systemd user instannce:
+
+[source,bash]
+----
+$ ssh sleeper@192.168.x.y
```

Comment:
I'd be happier if we can show the same through `ssh core@...`.

Reply:
OK, I'll try that